release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
+
+The upgrade to Kubernetes v1.22 removed the following deprecated
+`v1beta1` APIs. Migrate manifests and API clients to use the `v1` API
+version. For more information on migrating deprecated APIs, see the
+https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes
+documentation].
+
+.`v1beta1` APIs removed from Kubernetes v1.22
+[cols=",,",options="header",]
+|===
+|Feature |API |Notable Changes
+|Webhook resources
```

Comment:
Right now, your table uses a mix of generic terms (webhook resources), and actual object names (PriorityClass). By using generic terms, it might not actually be clear what falls under that category. Since these APIs are tied to direct objects, I’d recommend using the official object names (in monospace) in the first column, and repeating the API in the next column as necessary. This is what I see from the Kube page: 

- MutatingWebhookConfiguration
- ValidatingWebhookConfiguration
- CustomResourceDefinition
- APIService
- TokenReview                                      // <- This one seems to be missing entirely
- LocalSubjectAccessReview
- SelfSubjectAccessReview
- SubjectAccessReview
- CertificateSigningRequest
- Lease
- Ingress
- IngressClass
- ClusterRole
- ClusterRoleBinding
- Role
- RoleBinding
- PriorityClass
- CSIDriver
- CSINode
- StorageClass
- VolumeAttachment

OR, you could start with the API in the first column, and list several affected objects in the second column, like:

```
|`admissionregistration.k8s.io/v1beta1`
|`MutatingWebhookConfiguration`
`ValidatingWebhookConfiguration`
|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#webhook-resources-v122[Yes]
```

Either way, I'd also alphabetize by the first column.